### PR TITLE
Allow HTTP logging to be controlled

### PR DESCRIPTION
**Pull request by otoolep**
This change also re-enables DQ testing, as well as remove a bogus `warn` statement.

### CODE REVIEW

**Comment by otoolep** [comment]
@corylanou @jwilder

File: httpd/handler.go [hunk 1]
```diff
@@ -52,8 +52,9 @@ type Handler struct {
 	requireAuthentication bool
 	version               string
 
-	Logger     *log.Logger
-	WriteTrace bool // Detailed logging of write path
+	Logger            *log.Logger
+	HTTPAccessLogging bool // Log every HTTP access.
```
Comment (corylanou) [inline httpd/handler.go]: Can we just make this something like `EnableLogging` or `LoggingEnabled`?  This name is confusing when I don't see it in context.

**Comment by otoolep** [comment]
I also increased the interval between querying the server during testing to 100ms. I originally suggested 10ms, may mean less strain on the Circle container, and it should not actually make a difference to how long tests take to run.

File: cmd/influxd/config.go [hunk 2]
```diff
@@ -239,6 +240,10 @@ func NewConfig() *Config {
 	c.Snapshot.BindAddress = DefaultSnapshotBindAddress
 	c.Snapshot.Port = DefaultSnapshotPort
 
+	c.Logging.HTTPAccess = true
```
Comment (corylanou) [inline cmd/influxd/config.go]: There is an HTTPAPI section.  Should we put logging under that?  This logging section is starting to feel out of place.  Each are could have it's own logging flag now if we wanted.

**Comment by otoolep** [comment]
Integration test output is so much easier to work with now, I do think we should merge this (or something close to it).

File: httpd/handler.go (hunk 1 above)
Comment (otoolep) [inline httpd/handler.go]: Sure.

File: cmd/influxd/config.go (hunk 2 above)
Comment (otoolep) [inline cmd/influxd/config.go]: Possibly. I could go either way. Would you agree to punting on that?

Comment (otoolep) [inline cmd/influxd/config.go]: Distinct PR, perhaps?

**Comment by otoolep** [comment]
Updated with change to HTTP logging flag.

**Comment by corylanou** [comment]
@otoolep I'm fine with it in current state.  Our config might need some re-org at some point in the future anyway.

I do agree this was very much needed.  It was getting impossible to look through the failed CI output.  Thanks!

**Comment by otoolep** [comment]
Also fixes 2245.

**Comment by otoolep** [comment]
Thanks @corylanou -- CircleCI runs already looking more stable on this branch.

File: cmd/influxd/config.go (hunk 2 above)
Comment (jwilder) [inline cmd/influxd/config.go]: Agree w/ @corylanou.

**Comment by otoolep** [comment]
Fixes issue #2302

**Comment by otoolep** [comment]
Closed in favour of https://github.com/influxdb/influxdb/pull/2320

**Pull request closed without merging**